app/views/applications/show.html.erb

```diff
@@ -1,7 +1,22 @@
-<p>Name: <%= @applications.name %>
-<br>Full Address: <%= @applications.address %>, <%= @applications.city %>, <%= @applications.state %> <%= @applications.zip %>
-<br>Description: <%= @applications.description %>
-<br>All Pets: <% @applications.pets.each do |pet| %>
+<p>Name: <%= @application.name %>
+<br>Full Address: <%= @application.address %>, <%= @application.city %>, <%= @application.state %> <%= @application.zip %>
+<br>Description: <%= @application.description %>
+<br>All Pets: <% @application.pets.each do |pet| %>
   <%= pet.name %>
 <% end %>
-<br>Application Status: <%= @applications.status %></p>
+<br>Application Status: <%= @application.status %></p>
+
+<% if @application.status == "In Progress" %>
+  <%= "Add a Pet to this Application" %>
+  <%= form_with url: "/applications/#{@application.id}", method: :get, local: true do |form| %>
+    <br> <%= form.label :pet_name %>
+    <%= form.text_field :pet_name %>
+
+    <br> <%= form.submit "Submit" %>
```

Comment:
It behaves more like a search bar, so I would suggest changing "submit" to "search".

Reply:
I thought of that too, in the user story it said submit so thats why I went with submit but I would be up for changing it to search

Reply:
I think that we were following the descriptions listed in the user story for naming buttons and sections of the form